data/json/items/gun/shot.json

```diff
@@ -186,7 +182,7 @@
     "durability": 6,
     "loudness": 25,
     "clip_size": 6,
-    "reload": 100,
+    "reload": 200,
```

Comment:
Is that implausible for a revolver welded to a shotgun barrel and then modified to take shotgun cartridges? Presumably more difficult than reloading a factory specification pistol?